sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java

```diff
@@ -0,0 +1,60 @@
+package com.microsoft.appcenter.storage;
+
+import com.microsoft.appcenter.utils.storage.SharedPreferencesManager;
+import com.google.gson.Gson;
+import com.microsoft.appcenter.storage.models.TokenResult;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+
+public class TokenManager {
+    private final Gson gson; // TODO use Gson object from shared module.

```

Comment:
nit: Don't use TODOs inline, move above

Reply:
used this format from distribution module :see_no_evil: